scripts/rag_evaluation/create_evaluation_dataset/chunks_flatten_strategy/create_chunks.py

```diff
@@ -0,0 +1,63 @@
+import json
+import numpy as np
+from typing import List
+import matplotlib.pyplot as plt
+
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+
+from settings import OPENAI_MODEL_EMBEDDING
+
+
+text_splitter = RecursiveCharacterTextSplitter(
+    separators=["\n\n", "\n", " ", ""],
+    chunk_size=500,
+    chunk_overlap=50,
```

Comment:
These are relevant parameters which should also be stored in the experiment tracker, which also means that you should pass them in as parameters